### DISCUSSION TITLE
Official 1.0.0 version

### DISCUSSION THREAD

**Discussion by ragaller** [joeljeremy7/externalized-properties#176] [General]
Will there be an "official" 1.0.0 version in the near future?

**Comment by S-Kerrigan** [comment]
Yes, we can asking your, please, publishing distribution to Maven Repository?
I wish the update, because version has not updated since 2022

**Comment by joel-jeremy** [comment]
Hey there 👋 Sorry for the super late response! I'm glad you liked the library and found it useful! I'm working on releasing the 1.0.0, I'm just encountering some issues with Nexus but should be resolved soon!

**Comment by ragaller** [comment]
Good news! Thanks for your effort. Looking foward to the upcoming release! 👏

**Comment by joel-jeremy** [comment]
@ragaller @S-Kerrigan Happy to announce that v1.0.0 is now released! :)
